### PR TITLE
Validation: description-missing fixes.

### PR DESCRIPTION
**Pull request by brittmsantos**
Added missing descriptions to fix validation errors.

### CODE REVIEW

File: BingMaps/v8-web-control/map-control-concepts/test-data-generator-example.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Test Data Generator Example | Microsoft Docs"
+description: Provides a code example that creates two random polygons with a scaleFactor of 0.5 random polygon options, and a hole.
```
Comment (dereklh7) [inline BingMaps/v8-web-control/map-control-concepts/test-data-generator-example.md:3]: Comma is unnecessary here.

**Comment by brittmsantos** [comment]
@stevemunk @rbrundritt

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

File: BingMaps/v8-web-control/map-control-concepts/user-location/index.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "User Location | Microsoft Docs"
+description: Describes user location and provides a list of the PositionOptions Object, Position Object, and Coordinate Object properties.
```
Comment (stevemunk) [inline BingMaps/v8-web-control/map-control-concepts/user-location/index.md:3]:
```suggestion
description: Describes user location and provides a list of the PositionOptions Object, Position Object, and Coordinate Object properties with links to examples.
```

File: BingMaps/v8-web-control/modules/autosuggest-module/address-object.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Address Object (Autosuggest Module) | Microsoft Docs"
+description: Describes the address object, which represents a structured address, and provides a list of related properties.
```
Comment (stevemunk) [inline BingMaps/v8-web-control/modules/autosuggest-module/address-object.md:3]:
```suggestion
description: Describes the address object, which represents a structured address, and descriptions of its properties.
```

File: BingMaps/v8-web-control/modules/autosuggest-module/autosuggestoptions-object.md [hunk 4]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "AutosuggestOptions Object | Microsoft Docs"
+description: Describes the AutosuggestOptions object, which controls autosuggest options, and provides a list of the object's properties.
```
Comment (stevemunk) [inline BingMaps/v8-web-control/modules/autosuggest-module/autosuggestoptions-object.md:3]:
```suggestion
description: Describes the AutosuggestOptions object, which controls autosuggest options, and descriptions of each of its properties.
```

File: BingMaps/v8-web-control/modules/autosuggest-module/index.md [hunk 5]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Autosuggest Module | Microsoft Docs"
+description: Describes the Autosuggest module, which provides a list of suggested addresses or places, and provides lists of API reference articles and examples.
```
Comment (stevemunk) [inline BingMaps/v8-web-control/modules/autosuggest-module/index.md:3]:
```suggestion
description: The overview page for the Autosuggest Module section contains a description and links to the API reference articles and examples.
```

**Review by stevemunk — CHANGES_REQUESTED** [review]

**Comment by brittmsantos** [comment]
@stevemunk 

I've implemented the requested changes. Please let me know if there are any other issues preventing approval and merge. Thank you, and have a great day!

**Pull request merged**